### ISSUE TITLE
Mouse clicks are not recognized

### ISSUE THREAD

**Issue by JauernigIT** [Norbert515/detective#6]
I found that mouse clicks on my MacBook are not recognized, at least partially. Only approx. every 10th click leads to an action. This behavior applies for all areas of the application, Class/Cubit selection as well as node selection on the object tree etc.. Keyboard is working without troubles.

detective version
- in About: 1.0.0 (1)
- inside the app screen: v0.0.17

macOS 11.2

**Comment by Norbert515** [comment]
Should be fixed with v0.0.18 now. If the bug still occurs, feel free to reopen it.

**Comment by Norbert515** [comment]
Reopening as I just encountered it again. This seems to be a Flutter + macOS specific issue that is occurring randomly.

https://github.com/flutter/flutter/issues/76325

https://github.com/flutter/flutter/issues/70375

If anybody can reliably reproduce this, it would help a lot!

**Comment by Feichtmeier** [comment]
> 
> 
> Reopening as I just encountered it again. This seems to be a Flutter + macOS specific issue that is occurring randomly.
> 
> [flutter/flutter#76325](https://github.com/flutter/flutter/issues/76325)
> 
> [flutter/flutter#70375](https://github.com/flutter/flutter/issues/70375)
> 
> If anybody can reliably reproduce this, it would help a lot!

Nope, can reproduce this on Linux and Windows with navigation rails. After clicking it for 5 times or so, I have this error, too :(
